src/fancyline/tty/vt100.cr

```diff
@@ -10,11 +24,18 @@ class Fancyline
         super()
       end
 
-      # FIXME: Properly get the terminal dimensions, don't rely on environment
-      #        variables that are outdated at the time of reading them.
+      def winsize
```

Comment:
Not super important for this PR:

How expensive (in time) is this method?  It's going to be called a couple of times on every render.  On the long run maybe this could be cached for a short period of time.

Reply:
```text
ioctl  521.9k (  1.92µs) (± 7.16%)  8.80× slower
  env    4.6M (217.62ns) (± 5.06%)       fastest
```

Reply:
As expected, but nothing to worry about for this PR.